server/src/main/java/org/apache/druid/segment/metadata/AbstractSegmentMetadataCache.java

```diff
@@ -385,55 +329,26 @@ private void startCacheExec()
     );
   }
 
+  private void setInitializedAndReportInitTime(Stopwatch stopwatch)
+  {
+    // report the cache init time
+    if (initialized.getCount() == 1) {
+      long elapsedTime = stopwatch.elapsed(TimeUnit.MILLISECONDS);
+      emitter.emit(ServiceMetricEvent.builder().setMetric("metadatacache/init/time", elapsedTime));
+      log.info("%s initialized in [%,d] ms.", getClass().getSimpleName(), elapsedTime);
+      stopwatch.stop();
+    }
+    initialized.countDown();
+  }
+
   @LifecycleStart
   public void start() throws InterruptedException
   {
+    log.info("%s waiting for initialization.", getClass().getSimpleName());
     startCacheExec();
 
     if (config.isAwaitInitializationOnStart()) {
```

Comment:
I wanted this message to be logged irrespective of the `isAwaitInitializationOnStart` config, since this config is `false` by default on Coordinator. 
I am changing the message to `%s starting cache initialization.`